lambda/joberrorhandler/lambda_function.py

```diff
@@ -0,0 +1,28 @@
+import os
+import json
+import boto3
+
+TABLE_NAME =os.environ['DOCUMENTS_TABLE']
+
+def get_document_status(document_id):
+    d_client = boto3.client('dynamodb')
+    d_resposne = d_client.get_item(TableName=TABLE_NAME,Key={'documentId':{'S':str(document_id)}})
+    if d_resposne.get('Item') is not None:
+        if d_resposne['Item'].get('documentStatus') is not None:
+            return d_resposne['Item']['documentStatus'].get('S')
+    
+def update_document_status_to_failed(document_id):
+    d_client = boto3.client('dynamodb')
```

Comment:
nitpick: Why not move this client creation to the top of the file, or at least in the main handler function? Reduces repetition.

Reply:
Good point. Made the change